src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form method="post"
+    action="index.html"
+    class="form"
+    >
+      <input
+      class="big__form"
```

Comment:
lets change classname to `input`, and for small input add modifire `--small`
```suggestion
      class="input"
```